### PR TITLE
Istio Blog Post for 0.1 announcement.

### PR DESCRIPTION
**Pull request by smawson**
For now this is using today as the date, in order to be able to see what it looks like. I'll send a PR after this is in to change the date to the 24th.

### CODE REVIEW

File: _posts/2017-05-23-0.1-announcement.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Introducing Istio: A robust service mesh for microservices"
+overview: Istio 0.1 announcement
+published: true
+permalink: blog/istio-service-mesh-for-microservices.html
```
Comment (geeknoid) [inline _posts/2017-05-23-0.1-announcement.md]: Permalink should somehow contain the version number. This permalink won't be so hot as-is in two years when we're at version 3.0.

File: _posts/2017-05-23-0.1-announcement.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+attribution: The Istio Team
+layout: post
+type: markdown
+---
+
+Google, IBM, and Lyft are proud to announce the first public release of [Istio](https://istio.io/): an open source project that provides a uniform way to connect, secure, manage and monitor microservices. Our current release is targeted at the [Kubernetes](https://kubernetes.io/) environment; we intend to add support for other environments such as virtual machines and Cloud Foundry in the coming months.
```
Comment (geeknoid) [inline _posts/2017-05-23-0.1-announcement.md]: "Our current release..." that whole sentence should be down low in the doc. It should most certainly not be the second sentence of an announcement.

**Comment by richarddli** [comment]
Here's the link to the Datawire blog post: https://www.datawire.io/istio-and-datawire-ecosystem/.

File: _posts/2017-05-23-0.1-announcement.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+<!--end_excerpt-->
+
+## Background
+
+Writing reliable, loosely coupled, production-grade applications based on microservices can be challenging. As monolithic applications are decomposed into microservices, software teams have to worry about the challenges inherent in integrating services in distributed systems: they must account for service discovery, load balancing, fault tolerance, end-to-end monitoring, dynamic routing for feature experimentation, and perhaps most important of all, compliance and security.
+
+Inconsistent attempts at solving these challenges, cobbled together from libraries, scripts and StackOverflow snippets leads to solutions that vary wildly across languages and runtimes, have poor observability characteristics and can often end up compromising security.
```
Comment (geeknoid) [inline _posts/2017-05-23-0.1-announcement.md]: StackOverflow is two words -> Stack Overflow

File: _posts/2017-05-23-0.1-announcement.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Google, IBM and Lyft joined forces to create Istio from a desire to provide a reliable substrate for microservice development and maintenance, based on our common experiences building and operating massive scale microservices for internal and enterprise customers. Google and IBM have extensive experience with massive scale microservices in their own applications and with their enterprise customers in sensitive/regulated environments, while Lyft developed Envoy to address their internal operability challenges. [Lyft open sourced Envoy](https://eng.lyft.com/announcing-envoy-c-l7-proxy-and-communication-bus-92520b6c8191) after successfully using it in production for over a year to manage more than 100 services spanning 10,000 VMs, processing 2M requests/second.
```
Comment (geeknoid) [inline _posts/2017-05-23-0.1-announcement.md]: Doesn't sound good to repeat "massive scale" twice in a row.

File: _posts/2017-05-23-0.1-announcement.md (hunk 1 above)
Comment (smawson) [inline _posts/2017-05-23-0.1-announcement.md]: This is the permalink we've already told our partners we're using, so it is somewhat fixed already.

File: _posts/2017-05-23-0.1-announcement.md (hunk 2 above)
Comment (smawson) [inline _posts/2017-05-23-0.1-announcement.md]: This is the tl;dr portion of the blog post, we want to make it clear up front that we only support Kubernetes but will support others in the future.

File: _posts/2017-05-23-0.1-announcement.md (hunk 3 above)
Comment (smawson) [inline _posts/2017-05-23-0.1-announcement.md]: Done.

File: _posts/2017-05-23-0.1-announcement.md (hunk 4 above)
Comment (smawson) [inline _posts/2017-05-23-0.1-announcement.md]: Reworded.

**Comment by smawson** [comment]
Added Datawire blog post @richarddli, right now it is a 404, I assume that's OK.

File: _posts/2017-05-23-0.1-announcement.md (hunk 1 above)
Comment (rshriram) [inline _posts/2017-05-23-0.1-announcement.md]: You can set published to false right?

File: _posts/2017-05-23-0.1-announcement.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Introducing Istio: A robust service mesh for microservices"
```
Comment (rshriram) [inline _posts/2017-05-23-0.1-announcement.md]: Isn't this different from title on docs?

Comment (smawson) [inline _posts/2017-05-23-0.1-announcement.md]: That's the exact title in docs, what do you see?

**Comment by ldemailly** [comment]
too bad jenkins was down: there are bad links in the blog:

- ./_site/blog/istio-service-mesh-for-microservices.html
  *  External link https://blog.openshift.com/red-hat-istio-launch failed: 404 No error
  *  External link https://grpc.io failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: Peer certificate cannot be authenticated with given CA certificates
  *  External link https://weave.works/docs/tutorials/istio/istio failed: 404 No error
  *  External link https://www.datawire.io/istio-and-datawire-ecosystem failed: 404 No error
  *  External link https://www.projectcalico.org/welcoming-istio-to-the-kubernetes-networking-community failed: 404 No error
  *  internal image \img\istio_grafana_dashboard.png does not exist (line 1)
  *  internal image \img\istio_zipkin_dashboard.png does not exist (line 1)

**Comment by smawson** [comment]
FYI the new PR has link fixes (some are expected since they haven't been published), it will go in tomorrow morning.

**Comment by zinuga** [comment]
Weave Link
https://www.weave.works/blog/istio-weave-cloud

**Comment by zinuga** [comment]
add @googlegroups.com to the users and announce email groups at the end

**Comment by rettori** [comment]
the openshift post is scheduled to go live at 8am PT.

**Pull request merged**